README.md

```diff
@@ -86,6 +86,13 @@ The service provider metadata used to ease configuration of the SAML SP in the I
 * `:idp_sso_target_url` - The URL to which the authentication request should be sent.
   This would be on the identity provider. **Required**.
 
+* `:idp_slo_target_url` - The URL to which the single logout request and response should
+  be sent. This would be on the identity provider. Optional.
+
+* `:default_relay_state` - The URL to use as default RelayState on single logouts. The
```

Comment:
I'm not sure it's accurate to saw that the `RelayState` is a URL in all cases.

Also, can we code-format "RelayState" with backticks?